### ISSUE TITLE
Fix false remainder handling on privacy mask

### ISSUE THREAD

**Issue by MichaIng** [motioneye-project/motioneye#2404]
https://github.com/motioneye-project/motioneye/pull/2403#issuecomment-1089056122

**Comment by jmichault** [comment]
Is this bug already present  after PR <https://github.com/motioneye-project/motioneye/pull/2479>, which closed issue <https://github.com/motioneye-project/motioneye/issues/2434> ?

**Comment by MichaIng** [comment]
Yes, the underlying issue still exist, also the too large last row and column, depending on camera dimensions. I don't find much time at the moment, but let me keep this issue open to clean up this code block, when I do.
